### PR TITLE
Update app.scss

### PR DESCRIPTION
**Pull request by sultanrif**
Documentation

Fixes #

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by sultanrif** [comment]
Great code

**Review by ayushontop — COMMENTED** [review]
Why ?

**Comment by sultanrif** [comment]
Documentation

**Comment by ayushontop** [comment]
This is a unnecessary change .

File: app/styles/app.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/*importing all required libraries and file */
```
Comment (maze-runnar) [inline app/styles/app.scss:1]: until you didn't write that, no one knew that @import, import libraries, required. Add this comment in like 400 more files to it more clear.

**Pull request closed without merging**